models/demos/t3000/llama2_70b/tt/llama_attention_optimized.py

```diff
@@ -350,10 +360,26 @@ def attn_selfout(
 
         return tt_matmul_out_tensor
 
-    def prefill_forward(self, xs, rot_mats, user_id: int = 0, page_table=None, kv_cache=None):
+    def prefill_forward(
+        self,
+        xs,
+        rot_mats,
+        user_id: int = 0,
+        page_table=None,
+        kv_cache=None,
+        chunk_page_table=None,
```

Comment:
Good point. chunk paged table preparation has to happen somewhere. Since chunked prefill is only accessible from llama_generation, I decided to put the preparation there